specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/loadBalancer.json

```diff
@@ -800,6 +801,100 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/loadBalancers/{loadBalancerName}/outboundRules": {
+      "get": {
+        "tags": [
+          "LoadBalancers"
+        ],
+        "operationId": "LoadBalancerOutboundRules_List",
+        "description": "Gets all the outbound rules in a load balancer.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "loadBalancerName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the load balancer."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
You should include a 'default' response for all error cases, example: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/msi/resource-manager/Microsoft.ManagedIdentity/preview/2015-08-31-preview/ManagedIdentity.json 

          "default": {
            "description": "Error response describing why the operation failed.",
            "schema": {
              "$ref": "#/definitions/CloudError"
            }
          }

This should be addressed across your entire Swagger spec though. Please include this in your next PR.

Reply:
Yes, I will address it in the next PR -  Tracking item - https://trello.com/c/HrzSc9bZ/69-include-a-default-response-for-all-error-cases